build.gradle

```diff
@@ -167,3 +167,31 @@ curse {
     changelog = System.getenv("CHANGELOG").equals("none") ? getChangelogText() : System.getenv("CHANGELOG")
     releaseType = project.hasProperty('mod_appendix') ? "${mod_appendix}".toString() : 'release'
 }
+
+task checkTranslations << {
+    Map<String, String> mapen = new HashMap<String, String>()
+    (new File('resources/assets/enderio/lang/en_US.lang')).eachLine {
+        def (value1, value2) = it.tokenize( '=' )
+        if (value1 == null || value2 == null) {return}
+        mapen.put(value1, value2)
+    }
+    
+    for ( lang in ['cs_CZ', 'de_DE', 'fr_FR', 'ko_KR', 'pl_PL', 'pt_BR', 'ru_RU', 'zh_CN', 'zh_TW'] ) {
```

Comment:
I would much prefer a wildcard *.lang selector instead of hardcoded names, I know we don't get new translations often, but when we do it'll break.